Client/Frontend/Browser/URLBarView.swift

```diff
@@ -467,9 +462,8 @@ class URLBarView: UIView {
             self.rightBarConstraint?.updateOffset(URLBarViewUX.URLBarCurveOffset + URLBarViewUX.URLBarCurveBounceBuffer + tabsButton.frame.width)
 
             // Make the editable text field span the entire URL bar, covering the lock and reader icons.
-            self.locationTextField.snp_remakeConstraints { make in
+            self.locationTextField.snp_updateConstraints { make in
                 make.leading.equalTo(self.locationContainer).offset(URLBarViewUX.LocationContentOffset)
-                make.top.bottom.trailing.equalTo(self.locationContainer)
```

Comment:
because i set it initially and never change it, since we did remake, we had to always set it again? is that why?